README.adoc

```diff
@@ -79,7 +80,8 @@ cd start
 {: codeblock}
 endif::[]
 
-You can find the starting Java project in the `start` directory. It is a multi-module Maven project that is made up of the `system` and `inventory` microservices. Each microservice lives in its own corresponding directory, `system` and `inventory`.
+You can find the starting Java project in the `start` directory. 
+This project is a multi-module Maven project that is made up of the `system` and inven`tory microservices. Each microservice is located in its own corresponding directory, `system` and `inventory`.
```

Comment:
the `system` and inven`tory microservices

-->

the `system` and `inventory` microservices